raiden/network/pathfinding.py

```diff
@@ -176,11 +180,7 @@ def configure_pfs_or_exit(
         click.secho(msg)
         log.info("Using PFS", pfs_info=pathfinding_service_info)
 
-    return PFSConfiguration(
-        url=pfs_address,
-        eth_address=pfs_eth_address,
-        fee=pathfinding_service_info.get("price_info", 0),
-    )
+    return PFSConfiguration(url=pfs_address, eth_address=pfs_eth_address, fee=fee)
```

Comment:
With the fee > 0 changes, you should make `PFSConfiguration.eth_address` `Optional`.